### PR TITLE
Fix broken link of IntelliJ IDEA Setup Guide

### PR DESCRIPTION
**Pull request by gabrywu**
### _Why are the changes needed?_
close https://github.com/apache/incubator-kyuubi/issues/2511
fix link of IntelliJ IDEA Setup Guide

### _How was this patch tested?_
- [x] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [x] Add screenshots for manual tests if appropriate

- [x] [Run test](https://kyuubi.apache.org/docs/latest/develop_tools/testing.html#running-tests) locally before make a pull request

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -29,11 +29,14 @@ We hope that this will be a pleasant first experience for you to return to conti
 
 ## Get Involved
 
-In the process of using Apache Kyuubi, if you have any questions, suggestions, or improvement ideas, you can participate in the Kyuubi community building through the following suggested channels.
+In the process of using Apache Kyuubi, if you have any questions, suggestions, or improvement ideas, you can participate
+in the Kyuubi community building through the following suggested channels.
```
Comment (turboFei) [inline CONTRIBUTING.md:33]: not needed

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -29,11 +29,14 @@ We hope that this will be a pleasant first experience for you to return to conti
...
 
-- Join the [Mailing Lists](https://kyuubi.apache.org/mailing_lists.html) - the best way to keep up-to-date with the community.
+- Join the [Mailing Lists](https://kyuubi.apache.org/mailing_lists.html) - the best way to keep up-to-date with the
+  community.
```
Comment (turboFei) [inline CONTRIBUTING.md:36]: ditto

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -29,11 +29,14 @@ We hope that this will be a pleasant first experience for you to return to conti
...
 - [Issue Tracker](https://kyuubi.apache.org/issue_tracking.html) - tracking bugs, ideas, plans, etc.
-- [Github Discussions](https://github.com/apache/incubator-kyuubi/discussions) - second to mailing list for anything else you want to share or ask
+- [Github Discussions](https://github.com/apache/incubator-kyuubi/discussions) - second to mailing list for anything
+  else you want to share or ask
```
Comment (turboFei) [inline CONTRIBUTING.md:39]: ditto

**Review by turboFei — CHANGES_REQUESTED** [review]

**Review by turboFei — APPROVED** [review]

**Comment by turboFei** [comment]
cc @pan3793 

do we have plan to unify the docs for latest version and master unreleased version?

**Review by hddong — APPROVED** [review]

**Comment by turboFei** [comment]
thanks, merged to master

**Pull request closed without merging**